### ISSUE TITLE
could not find a copy of eslint to link

### ISSUE THREAD

**Issue by Moxie1776** [vuetifyjs/eslint-config-vuetify#7]
When running `yarn add eslint-config-vuetify -D ` I get the following error

error An unexpected error occurred: "could not find a copy of eslint to link in /home/jeff/wexite/client/node_modules/eslint-config-vuetify/node_modules".

Here are the dependencies from my package.json

```
"dependencies": {
    "@nuxtjs/auth": "^4.9.1",
    "@nuxtjs/axios": "^5.11.0",
    "@nuxtjs/dotenv": "^1.4.1",
    "echarts": "^4.8.0",
    "nuxt": "^2.13.0",
    "typescript": "^3.9.5",
    "vee-validate": "^3.3.4",
    "vue": "^2.6.11",
    "vue-echarts": "^5.0.0-beta.0"
  },
  "devDependencies": {
    "@nuxtjs/eslint-config": "^3.0.0",
    "@nuxtjs/eslint-module": "^2.0.0",
    "@nuxtjs/vuetify": "^1.11.2",
    "debug": "^4.1.1",
    "eslint": "^7.2.0",
    "eslint-config-prettier": "^6.10.0",
    "eslint-plugin-nuxt": ">=0.4.2",
    "eslint-plugin-prettier": "^3.1.2",
    "prettier": "^2.0.5",
    "vue-template-compiler": "^2.6.11",
    "webpack": "^4.43.0"
  }
```

**Comment by gotenxds** [comment]
Same here, this happens in 0.6.1, works correctly in 0.6.1

**Comment by nelsonlarocca** [comment]
happens to me too

**Comment by KaelWD** [comment]
You might be looking for https://github.com/vuetifyjs/eslint-plugin-vuetify, this config is for consistent formatting between our internal projects.
